v0/destinations/facebook_pixel/data/FBPIXELUserDataConfig.json

```diff
@@ -0,0 +1,152 @@
+[
+  {
+    "destKey": "external_id",
+    "sourceKeys": [
+      "userId",
+      "traits.userId",
+      "traits.id",
+      "context.traits.userId",
+      "context.traits.id",
+      "anonymousId"
+    ],
+    "required": true,
+    "metadata": {
+      "type": "hashToSha256"
```

Comment:
Hashing was/is optional on this param, this could cause problems for people migrating who did not used to hash this key (for instance, audiences with internal uuid)